src/config/dynamic-theme-fixes.config

```diff
@@ -4571,6 +4571,29 @@ CSS
 
 ================================
 
+lingvoforum.net
+
+INVERT
+div#header
+div#header img.avatar
+span.topslice
+span.botslice
+
+CSS
+body {
+  background-image: none;
+}
+
+div#header {
+    color: black;
```

Comment:
Change it to 
```
color: ${white} !important;
```

Reply:
!important added; unfortunately neither ${white} nor ${blank} work correct, one of them gives unreadable text in dark scheme, other in light scheme. Only ugly "black" works in both schemes.